handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -161,6 +162,7 @@ public final class OpenSsl {
             }
 
             IS_BORINGSSL = "BoringSSL".equals(versionString());
+            IS_AWSLC = versionString().startsWith("AWS-LC");
             if (IS_BORINGSSL) {
                 EXTRA_SUPPORTED_TLS_1_3_CIPHERS = new String [] { "TLS_AES_128_GCM_SHA256",
```

Comment:
AWS-LC includes these ciphers in `SSL_get_ciphers()` ?

Reply:
Correct, we made this modification back in January of this year to match the OpenSSL behavior of returning TLS 1.3 and TLS 1.2 and below cipher suites in the list.